src/main/java/com/almightyalpaca/discord/jdabutler/util/WebhookAppender.java

```diff
@@ -0,0 +1,76 @@
+package com.almightyalpaca.discord.jdabutler.util;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.AppenderBase;
+import ch.qos.logback.core.encoder.Encoder;
+import ch.qos.logback.core.status.ErrorStatus;
+import net.dv8tion.jda.core.utils.SimpleLog;
+import net.dv8tion.jda.webhook.WebhookClient;
+import net.dv8tion.jda.webhook.WebhookClientBuilder;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class WebhookAppender extends AppenderBase<ILoggingEvent> {
+    private static Pattern WH_PATTERN = Pattern.compile("(?:https?://)?(?:\\w+\\.)?discordapp\\.com/api(?:/v\\d+)?/webhooks/(\\d+)/([\\w-]+)(?:/(?:\\w+)?)?");;
+
+    private Encoder<ILoggingEvent> encoder;
+    private String webhookUrl;
+    private WebhookClient client;
+
+    @Override
+    public void start() {
+        int warn = 0;
+        if(encoder == null) {
+            addStatus(new ErrorStatus("No encoder specified", this));
+            warn++;
+        }
+        Matcher matcher = null;
+        if(webhookUrl == null || webhookUrl.isEmpty()) {
+            addStatus(new ErrorStatus("No Webhook url specified", this));
+            warn++;
+        } else {
+            matcher = WH_PATTERN.matcher(webhookUrl);
+            if(!matcher.matches()) {
+                addStatus(new ErrorStatus("Webhook url was not a valid Webhook url", this));
+                warn++;
+            }
+        }
+        if(warn == 0) {
+            client = new WebhookClientBuilder(Long.parseUnsignedLong(matcher.group(1)), matcher.group(2)).setDaemon(true).build();
```

Comment:
Remind me to change this stuff to JDA's new WebhookClientBuilder constructor that's coming with the hacktoberfest branch.